app/views/onboarding/NotificationDetails.tsx

```diff
@@ -1,18 +1,15 @@
 import React from 'react';
+import { useNavigation } from '@react-navigation/native';
 import { useTranslation } from 'react-i18next';
-import { isGPS } from '../../COVIDSafePathsConfig';
 
+import { isGPS } from '../../COVIDSafePathsConfig';
 import { useStrategyContent } from '../../TracingStrategyContext';
 import ExplanationScreen, { IconStyle } from '../common/ExplanationScreen';
+import { Screens } from '../../navigation';
 
-interface NotificationDetailsProps {
-  navigation: any;
-}
-
-const NotificationDetails = ({
-  navigation,
-}: NotificationDetailsProps): JSX.Element => {
+const NotificationDetails = (): JSX.Element => {
```

Comment:
👏  I think I saw a couple of warnings on `app/views/onboarding/PersonalPrivacy.tsx` and `app/views/onboarding/ShareDiagnosis.tsx` that are related to this. Is there a place where we could ask people to use the hook or the props from the library instead of typing it with any?

Reply:
Perhaps, I think a lot of this is coming from having ts being introduced gradually into the code base.  Imo, the best way to socialize not using any is via pairing and code review.  We do have a linting rule for no-explicit-any which we could up to error if we want?